pom.xml

```diff
@@ -51,6 +51,7 @@
         <module>jte-runtime-cp-test</module>
         <module>jte-test-report</module>
         <module>jte-deploy-nexus</module>
+        <module>jte-gradle-plugin</module>
```

Comment:
I think we should move the module up to where the maven module is.

I'm not sure if it will be deployed to nexus correctly, when it is after `jte-deploy-nexus`.

Reply:
Sure, I didn't know the details.